.gitignore

```diff
@@ -35,3 +35,4 @@ celerybeat-schedule
 
 # Ignore coverage results
 coverage
+coverage.xml 
```

Comment:
Add `.coverage` in here. Remove the `.coverage` file that is included in this PR.